docs/src/main/sphinx/release/release-422.md

```diff
@@ -0,0 +1,49 @@
+# Release 422 (12 Jul 2023)
+
+## General
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18005`)
+* Fix queries hanging when the `task.retry-policy` configuration property is set
+  to `TASK` and other worker node failures occur. ({issue}`18175 `)
+
+## Security
+
+* Add OAuth 2.0 support for validating JWT types. ({issue}`17640`)
+* Fix error when the `http-server.authentication.type` configuration property
+  is set to `oauth2` or `jwt` and the `principal-field` property has a different
+  value. ({issue}`18210`)
+
+## BigQuery connector
+
+* Add support for writing to columns with a `timestamp with time zone` type. ({issue}`17793`)
+
+## Delta Lake connector
+
+* Add support for renaming columns. ({issue}`15821`)
+* Improve performance of reading from tables with a large number of
+  [checkpoints](https://docs.delta.io/latest/delta-batch.html#-data-retention). ({issue}`17405`)
+* Disallow using the `vacuum` procedure when the max writer version is above 5. ({issue}`18095`)
```

Comment:
what writer? what does this mean for a user?  Using a newer version of Databricks or something?